libs/architectures/mlpe/architectures/embeddings/dense.py

```diff
@@ -54,18 +54,22 @@ class NChannelDenseEmbedding(nn.Module):
 
     def __init__(
         self,
-        N,
-        in_shape,
-        out_shape,
+        shape: Tuple[int, int],
+        context_dim: int,
         **kwargs,
     ) -> None:
         super().__init__()
-        self.N = N
+        n_ifos, in_features = shape
         self.activation = kwargs.get("activation", torch.relu)
```

Comment:
Would you remove this line @EthanMarx. I probably put it there, I don't know what why. `activation` does not need a special default.